web/vite.config.ts

```diff
@@ -26,7 +26,11 @@ const config = (mode: string) => {
 				'/hasura': {
 					target: 'http://localhost:4000/console'
 				}
-			}
+			},
+			// Serve rocket.glb from the public directory
+            fs: {
+                allow: ['./static/models']
+            }
```

Comment:
I thought it was already being served since the static folder is already public :thinking:  do you get any errors without it?